.gitignore

```diff
@@ -5,4 +5,6 @@
 /dist
 /docs/_build
 /.coverage
-.idea
\ No newline at end of file
+.idea
+/venv
+*.vscode
```

Comment:
Keep editor preferences local to your setup (that is your $HOME/.gitignore), do not include them in the project.

Reply:
ACK :)

Reply:
Fixed.